server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');

```

Comment:
Consider using a more descriptive variable name instead of 'PORT'. For example, 'SERVER_PORT' could be more informative.